src/MCPClient.Dockerfile

```diff
@@ -99,4 +99,4 @@ RUN mkdir -p /etc/archivematica && (echo "---"; \
 
 USER archivematica
 
-ENTRYPOINT /src/MCPClient/lib/archivematicaClient.py
+ENTRYPOINT ["/src/MCPClient/lib/archivematicaSingleClient.py"]
```

Comment:
Two things are happening here:
- Use the `exec` form instead of the `shell` form so we can capture signals from the application.
- Use the new `archivematicaSingleClient.py` executable which is the new single-threaded implementation.